src/FindGuitarTester.java

```diff
@@ -20,7 +31,12 @@ public static void main(String[] args) {
     }
   }
 
-  private static void initializeInventory(Inventory inventory) {
+  /**
+   * this method will set start the inventory which the guitar
```

Comment:
Don't need this method.  Just say 'Starts the inventory which...' I like my methods to start with action words.